tsconfig.json

```diff
@@ -5,9 +5,6 @@
     "compilerOptions": {
         "baseUrl": "./",
         "outDir": "./dist",
-        "useDefineForClassFields": false,
-        "target": "ES2017",
```

Comment:
we need target `es2017` to build the library, `es2022` + `useDefineForClassField: false` only for demo

Reply:
extended by default from root config

<img width="354" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/f19e8463-44f4-487f-a127-847ad6d2363a">

<img width="292" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/db914425-7f48-43f8-a7e2-5f98ecbbd4dc">